build.sbt

```diff
@@ -56,7 +56,7 @@ scriptedSbt := {
   if (jdk >= 21)
     "1.9.0" // first release that supports JDK21
   else
-    (pluginCrossBuild / sbtVersion).value
+    "1.3.3" // get https://github.com/sbt/sbt/issues/1673 to avoid race conditions
```

Comment:
See https://github.com/scalacenter/sbt-scalafix/pull/411#issuecomment-2070919676. I can't tell for sure, but I think that storing analysis files slightly changed the timing of concurrent executions of `compile`, causing that new transient failure.